tests/consensus_tests/auth_tests/test_jwt_access.py

```
@@ -654,18 +654,18 @@ def test_all_grpc_endpoints_are_covered():
-    path: str,
-    body: Optional[Union[dict, Callable[[], dict]]],
-    should_succeed: bool,
-    token: str,
-    path_params: dict = {},
-    request_kwargs: dict = {},
+        method: str,
+        path: str,
+        body: Optional[Union[dict, Callable[[], dict]]],
+        should_succeed: bool,
+        token: str,
+        path_params: dict = {},
```

Ruff (0.14.1)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)